src/compiler/diagnosticMessages.json

```diff
@@ -7014,6 +7014,10 @@
         "category": "Error",
         "code": 9037
     },
+    "Computed property names on class or object literals cannot be inferred with --isolatedDeclarations.": {

```

Comment:
Does this diag need to be added to the fixer?

Reply:
Since it can have the same related span attached, I suppose so!